doc/source/default-settings.rst

```diff
@@ -165,6 +165,41 @@ General settings
 
     **Note** The older (misspelled) version ``"formater"`` is deprecated.
 
+.. papis-config:: slugify-lowercase
```

Comment:
```suggestion
.. papis-config:: document-name-lowercase
```
and same for the others? Suggestions welcome, but `slugify` seems very tied to using that one library to do the cleanup.

Reply:
I'm not sure 'document-name' is appropriate. I know the function is called that, but isn't it also used for paths? Maybe something like 'filepath'? Not the prettiest word, but at least it's descriptive...

Reply:
Yeah, I think it's pretty exclusively used for files and folders. `document-paths-lowercase`?

Reply:
I shortened the ``document`` to ``doc``because the option names got pretty long otherwise. I hope that is still okay with you.

Reply:
That's fine! Thanks for fixing it!